tests/test_playbooks/scc_product.yml

```diff
@@ -0,0 +1,16 @@
+---
+- hosts: tests
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+  - include: tasks/scc_product.yml
+    vars:
+      scc_product_name: 'SUSE Linux Enterprise Server'
+      scc_product_friendly_name: 'SUSE Linux Enterprise Server 10 SP1 online i386'
+      # requires synced scc_account named "test"
```

Comment:
I guess we cannot set that up, as it would require a working SUSE account?

But maybe we still can provide a boilerplate w/o a set username/password?

Reply:
> I guess we cannot set that up, as it would require a working SUSE account?
> 
> But maybe we still can provide a boilerplate w/o a set username/password?

You mean providing false scc_account and expect the module to fail or record on actual scc_account but only provide it's name in playbook?

Reply:
no, I mean provide a play against `localhost`, that would setup `scc_account` but feed it with wrong/no SCC credentials so that a re-record would fail unless the user configures it properly.

in a way like https://github.com/theforeman/foreman-ansible-modules/blob/29f903edb41ce4b07ea7c336c7e59f34efa84480/tests/test_playbooks/repository_set.yml#L10-L13

the manifest we have commited in the repo is not sufficient for this to work, but someone can provide a working one for re-recording.